Index: FileCopyUtils.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/FileCopyUtils.java b/FileCopyUtils.java
--- a/FileCopyUtils.java	(revision 93e8632db3aae444ae4b3c342580b16d9359afd0)
+++ b/FileCopyUtils.java	(date 1637689858737)
@@ -1,15 +1,13 @@
 package de.kimminich.pitfalls;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.io.IOException;
+import java.io.*;
 
 /**
  * Helper class for copying files with a JDK version <1.7
  */
 public class FileCopyUtils {
 
+    @edu.umd.cs.findbugs.annotations.SuppressWarnings({ "OBL_UNSATISFIED_OBLIGATION" })
     public static void copy(File from, File to) {
         try {
             copy(new FileInputStream(from), new FileOutputStream(to));
@@ -29,16 +27,17 @@
         } catch (IOException e) {
             e.printStackTrace();
         } finally {
-            try {
-                if (in != null) {
-                    in.close();
-                }
-                if (out != null) {
-                    out.close();
-                }
-            } catch (IOException ignore) {
-            }
+            close(in);
+            close(out);
+        }
+    }
+
+    private static void close(Closeable c) {
+        try {
+            if (c != null) {
+                c.close();
+            }
+        } catch (IOException ignore) {
         }
     }
-
 }
